mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+mypy_path = src
+explicit_package_bases = True
+namespace_packages = True
+
+[mypy-rasterio.*]
+ignore_missing_imports = True
+
+[mypy-shapely.*]
+ignore_missing_imports = True
```

Comment:
added these to avoid errors like `error: Skipping analyzing "rasterio": found module but no type hints or library stubs`